src/sources/h3-query-source.ts

```diff
@@ -59,7 +62,12 @@ export const h3QuerySource = async function (
   return baseSource<UrlParameters>('query', options, urlParameters).then(
     (result) => ({
       ...(result as TilejsonResult),
-      widgetSource: new WidgetQuerySource(options),
+      widgetSource: new WidgetQuerySource({
+        ...options,
+        // NOTE: passing redundant spatialDataColumn here to apply the default value 'h3'
+        spatialDataColumn,
```

Comment:
Hm, good catch here! I'm a little nervous we'll forget to do this for future default values, maybe we should refactor how the defaults are handled. But we can do that in another PR later, your comment is a good callout for now. 👍🏻